### PR TITLE
optimize: optimize partial code handling

### LINKED ISSUE #3690: `Optimize Request:` Improve code quality.（提升代码质量请求）
Use the `SonarLint` plugin, `FindBugs` plugin or other plugin to analysis the `seata` project.
There will be a lot of code quality problems, even bugs. I hope people who are interested can help optimize the code.

使用`SonarLint`插件、`FindBugs`插件或其他插件来分析`seata`项目源码。
会找出非常多的代码质量问题，甚至有可能是隐藏的BUG。希望有兴趣的同学能够帮助提升代码质量。
在seata的源码中找找茬，提PR参与社区共建。
非常感谢。

# the analysis report like this（分析报告如下）:
图片

### PR DESCRIPTION
**Pull request by pengten**
- [x] I have registered the PR [changes](https://github.com/seata/seata/tree/develop/changes).

### Ⅰ. Describe what this PR did
improve code quality

### Ⅱ. Does this pull request fix one issue?

fixes #3690 

### Ⅴ. Special notes for reviews
this is my first pull request，please

### CODE REVIEW

File: server/src/main/java/io/seata/server/session/AbstractSessionManager.java [hunk 1]
```diff
@@ -51,30 +51,30 @@
...
+    protected AbstractSessionManager() {
     }
 
     /**
      * Instantiates a new Abstract session manager.
      *
      * @param name the name
      */
-    public AbstractSessionManager(String name) {
+    protected AbstractSessionManager(String name) {
```
Comment (a364176773) [inline server/src/main/java/io/seata/server/session/AbstractSessionManager.java:62]: public

File: server/src/main/java/io/seata/server/session/AbstractSessionManager.java [hunk 2]
```diff
@@ -51,30 +51,30 @@
     /**
      * Instantiates a new Abstract session manager.
      */
-    public AbstractSessionManager() {
+    protected AbstractSessionManager() {
```
Comment (a364176773) [inline server/src/main/java/io/seata/server/session/AbstractSessionManager.java:54]: public

**Comment by pengten** [comment]
@a364176773 you are right! I have fixed it, please review again.

**Review by a364176773 — APPROVED** [review]
LGTM.  please register your personal and pr information in these two files 
https://github.com/seata/seata/blob/develop/changes/1.5.0.md
https://github.com/seata/seata/blob/develop/changes/en-us/1.5.0.md

**Comment by a364176773** [comment]
It would be more appropriate to change the title to 'optimize partial code handling'

**Comment by pengten** [comment]
> It would be more appropriate to change the title to 'optimize partial code handling'
done

**Comment by a364176773** [comment]
发现你本地的git没有绑定你现在的github邮箱,绑定一下,再提交一次,否则你的改动可能不会被正确记录

**Comment by pengten** [comment]
> 发现你本地的git没有绑定你现在的github邮箱,绑定一下,再提交一次,否则你的改动可能不会被正确记录

Thanks for reminding, i have corrected.

File: seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/main/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfiguration.java [hunk 3]
```diff
@@ -37,7 +37,6 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SpringCloudAlibabaConfiguration.class);
```
Comment (caohdgege) [inline seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/main/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfiguration.java:38]: 这一行也是可以删除的，一起改一下吧

**Review by caohdgege — APPROVED** [review]
LGTM

File: seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/main/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfiguration.java (hunk 3 above)
Comment (pengten) [reply seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/main/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfiguration.java]: of course

File: seata-spring-autoconfigure/seata-spring-autoconfigure-core/src/main/java/io/seata/spring/boot/autoconfigure/provider/SpringBootConfigurationProvider.java [hunk 4]
```diff
@@ -60,7 +60,7 @@ public Object intercept(Object proxy, Method method, Object[] args, MethodProxy
                     } else if (args.length == 2) {
```
Comment (slievrly) [inline seata-spring-autoconfigure/seata-spring-autoconfigure-core/src/main/java/io/seata/spring/boot/autoconfigure/provider/SpringBootConfigurationProvider.java:60]: args.length > 1

Comment (pengten) [reply seata-spring-autoconfigure/seata-spring-autoconfigure-core/src/main/java/io/seata/spring/boot/autoconfigure/provider/SpringBootConfigurationProvider.java:60]: I will fix it in the next PR

Comment (pengten) [reply seata-spring-autoconfigure/seata-spring-autoconfigure-core/src/main/java/io/seata/spring/boot/autoconfigure/provider/SpringBootConfigurationProvider.java:60]: > I will fix it in the next PR

#4262

**Pull request merged**